src/main/kotlin/lotto/domain/LottoPurchase.kt

```diff
@@ -12,14 +9,14 @@ class LottoPurchase {
         return LottoPrice(moneyToInt)
     }
 
-    fun getLotto(price: LottoPrice) = price.get() / 1000
+    fun getLottoCount(price: LottoPrice): Int = price / 1000
 
     fun getLottoTickets(count: Int): List<LottoTicket> {
```

Comment:
(이번 단계 요구사항이기도 한) 일급 컬렉션을 적용해 봅시다.
```suggestion
    fun getLottoTickets(count: Int): LottoTickets {
```
아래 내용을 참고해 주세요!
https://tecoble.techcourse.co.kr/post/2020-05-08-First-Class-Collection/